app/models/spree/store_decorator.rb

```diff
@@ -6,6 +6,6 @@
   private
 
   def build_default_configuration
-    build_braintree_configuration
+    build_braintree_configuration if SolidusPaypalBraintree::Configuration.table_exists?
```

Comment:
Interesting. Why do we need this? The table should be available if all migrations have been run

Reply:
We need this is because of an old migration that creates a Store.
This was later fixed in solidus: https://github.com/solidusio/solidus/pull/488/commits/fd5cac10c69ba8b651245a3e0008af523e474d21

Reply:
👍